sklearn/tests/test_kernel_approximation.py

```diff
@@ -81,16 +82,19 @@ def test_additive_chi2_sampler():
 def test_skewed_chi2_sampler():
     # test that RBFSampler approximates kernel on random data
 
-    # compute exact kernel
     c = 0.03
-    # set on negative component but greater than c to ensure that the kernel
-    # approximation is valid on the group (-c; +\infty) endowed with the skewed
-    # multiplication.
-    Y[0, 0] = -c / 2.
+
+    # approximate kernel mapping
+    transform = SkewedChi2Sampler(skewedness=c, n_components=1000,
```

Comment:
It's a bad sign if you have to change any tests, because that probably means that you changed behavior and are breaking someone's code.

Reply:
+1 don't do this.

Reply:
I agree, I forgot about those comments...  I just put the PR into WIP until there's a better solution (see comment above)